updates/julian_1.sql

```diff
@@ -0,0 +1,159 @@
+ALTER TABLE selektion_bewertung ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_bkz ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_datgenauigkeit ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_dmghband ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_echtheit ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_editor ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_ethnie ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_ethnienerhalt ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_funktion ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_geschlecht ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_grammatikgeschlecht ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_janein ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_kasus ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_konvent ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_lebendverstorben ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_ort ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_quellengattung ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_reihe ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_sammelband ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_stand ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_urkundeausstellerempfaenger ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE selektion_verwandtschaftsgrad ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE suche_favoriten ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE content ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE ueberlieferung_edition ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE urkunde ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE urkunde_betreff ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE urkunde_dorsalnotiz ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE urkunde_hataussteller ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE urkunde_hatempfaenger ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+#
+#
+#add a field to neg tables to associated migrated rows with dmp rows
+#
+#
+ALTER TABLE bemerkung ADD provenance_id VARCHAR (255) DEFAULT NULL;
```

Comment:
I think we should also add a UNIQUE constraint to every table here, for several reasons:
- consistency during execution of our import program (each ID should only exist once)
- Performance improvement when doing lookups

Reply:
fixed!